Build/Build.csproj

```diff
@@ -68,7 +68,7 @@
       <Private>True</Private>
```

Comment:
Hint path is still wrong.

Reply:
Also, need to check .BuildProject file.

The release process script might also require an update to msBuildPath in ReleaseProcessScript.config file since there's still MSBuild 12 targeted.